index.js

```diff
@@ -1,29 +1,23 @@
-const {
-  config: authentication,
-  befores = [],
-  afters = [],
-} = require('./authentication');
+/* eslint-disable global-require */
+
+const creates = require('./creates');
+const triggers = require('./triggers');
+const authentication = require('./authentication');
+const { befores, afters } = require('./middleware');
 
 module.exports = {
-  // This is just shorthand to reference the installed dependencies you have.
-  // Zapier will need to know these before we can upload.
   version: require('./package.json').version,
   platformVersion: require('zapier-platform-core').version,
 
   authentication,
-
   beforeRequest: [...befores],
-
   afterResponse: [...afters],
 
-  // If you want your trigger to show up, you better include it here!
-  triggers: {},
+  creates,
```

Comment:
So I see you went with a pattern of including an index.js file per action type (creates, triggers, etc). Nothing wrong with this approach.  I'm linking here an [example](https://gitlab.com/zapier/team-integrations/zapier-platform-integrations/-/blob/main/apps/google-forms/index.js#L23) of our normal approach in these cases to avoid creating multiple index.js files just for reference

Reply:
Ty, I think the index.js pattern reduces some noise and makes it easier to handle cases like the scaffold tool